static/src/js/components/GranuleFilters/__tests__/GranuleFiltersForm.test.js

```diff
@@ -973,22 +1036,27 @@ describe('GranuleFiltersForm component', () => {
           },
           values: {
             equatorCrossingDate: {
-              startDate: '2019-08-13T00:00:00:000Z',
+              startDate: '',
               endDate: '2019-08-14T23:59:59:999Z'
             }
           }
         })
-        const equatorCrossingDateSection = enzymeWrapper.find(SidebarFiltersItem).at(5)
-        equatorCrossingDateSection.find(TemporalSelection).prop('onSubmitStart')(moment('2019-08-13T00:00:00:000Z', 'YYYY-MM-DDTHH:m:s.SSSZ', true))
-
-        expect(props.setFieldTouched).toHaveBeenCalledTimes(1)
-        expect(props.setFieldTouched).toHaveBeenCalledWith('equatorCrossingDate.startDate')
-        expect(props.setFieldValue).toHaveBeenCalledTimes(1)
-        expect(props.setFieldValue).toHaveBeenCalledWith('equatorCrossingDate.startDate', '2019-08-13T00:00:00:000Z')
+        const equatorCrossingStartDateTextField = screen.getAllByRole('textbox', { name: 'Start Date' })[1]
+
+        await user.click(equatorCrossingStartDateTextField)
+        await user.type(equatorCrossingStartDateTextField, '2019-08-13')
+        await user.tab(equatorCrossingStartDateTextField)
+
+        expect(setFieldTouched).toHaveBeenCalledTimes(10)
```

Comment:
It looks like this was only called once previously. Did something change here? Or is that just an expected difference between enzyme and rtl?

Reply:
Yes so this difference is because I think wth Enzyme we are setting value at once. Typing it with RTL is more representative of how a user would interact with this. Its related to that issue we talked about where we are submitting the value on each type instead of on-blur

Reply:
This is the accompanying ticket for that [EDSC-4218](https://bugs.earthdata.nasa.gov/browse/EDSC-4218)